commons/src/main/java/org/infinispan/commons/util/concurrent/jdk8backported/Eviction.java

```diff
@@ -0,0 +1,44 @@
+package org.infinispan.commons.util.concurrent.jdk8backported;
+
+/**
+ * @author Will Burns
```

Comment:
Do we consider this as a public API? If so, could you please add some Javadoc (or remove it completely if we don't).